app/models/form526_submission.rb

```diff
@@ -126,9 +126,12 @@ def submit_with_birls_id_that_hasnt_been_tried_yet!(
     log_exception_to_sentry e, extra_content_for_sentry
   end
 
+  # Note that the User record is stored in Redis temporarily -- `User.redis_namespace_ttl`
```

Comment:
This comment makes it sounds like we plan to change this. I don't believe we do. Maybe the word "cache" would be better here.

Reply:
Updated.